src/node.rs

```diff
@@ -252,17 +260,18 @@ impl InMemoryNode {
         let execution_mode = TxExecutionMode::EthCall {
             missed_storage_invocation_limit: 1000000,
         };
+        let (mut block_context, block_properties) = create_test_block_params();
```

Comment:
use create_block_context() (as it sets stuff like l1_gas_price corretctly -- that create_test_block_params doesn't)